plan/main

```diff
@@ -1,9 +1,8 @@
 #include <turnkey/base>
 #include <turnkey/lamp>
 
-php5-gd
-php5-cli
-php5-curl
-php5-sqlite
-
+php7.0-gd
+php-cli
+php7.0-curl
+php7.0-mbstring
```

Comment:
as this is debian based, you'd probably better want to install the following packages instead:

- [php-gd](https://packages.debian.org/stretch/php-gd)
- [php-curl](https://packages.debian.org/stretch/php-curl)
- [php-mbstring](https://packages.debian.org/stretch/php-mbstring)

there is no need to tie to php 7.0. Yii is constantly developed to work with the latest php version.

Reply:
Good catch @cebe - @spaghettimaster can you please adjust these for me? Just commit and push to the same branch and it will auto appear here (apologies on the explanation if you knew that already).